src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -5,29 +5,51 @@ export const environmentProjectSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['name', 'type', 'enabled'],
+    description:
+        "Describes an environment as it's configured for the project it's connected to",
     properties: {
         name: {
             type: 'string',
+            example: 'development',
+            description: 'The name of the environment',
         },
         type: {
             type: 'string',
+            example: 'production',
+            description:
+                'The [type of environment](https://docs.getunleash.io/reference/environments#environment-types).',
         },
         enabled: {
             type: 'boolean',
+            example: true,
+            description:
+                '`true` if the environment is enabled for the project, otherwise `false`',
         },
         protected: {
             type: 'boolean',
+            example: false,
+            description:
+                '`true` if the environment is protected, otherwise `false`. A *protected* environment can not be deleted.',
         },
         sortOrder: {
-            type: 'number',
+            type: 'integer',
+            example: 1,
+            description:
+                'Priority of the environment in a list of environments, the lower value, the higher up in the list the environment will appear',
         },
         projectApiTokenCount: {
-            type: 'number',
-            nullable: true,
+            type: 'integer',
+            minimum: 0,
+            example: 5,
+            description:
+                'How many client and frontend API tokens that have access to this project',
```

Comment:
Great! I suggest rewording it to the suggestion below, but I'll leave it up to you.


```suggestion
                'The number of client and front-end API tokens that have access to this project',
```